package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "irontask",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
same in this and the package-lock. the version bump is done automatically by the build system